k8s/agent-resources.yaml

```diff
@@ -142,6 +144,8 @@ spec:
               mountPath: /var/log
             - name: vardata
               mountPath: /var/data
+            - name: varliblogdna
+              mountPath: /var/lib/logdna
```

Comment:
Do we still want to mount it after changing the default? should we document it and leave the yaml files as is?

Reply:
I checked with Albert, he'd like it enabled by default on kubernetes